### ISSUE TITLE
Description of pasted image should be empty by default

### ISSUE THREAD

**Issue by provokateurin** [nextcloud/text#4665]
**Is your feature request related to a problem? Please describe.**
When you past an image it will set `image.png` or `image (x).png` as the default description. This is not useful in my opinion as the file name is irrelevant (especially because it is autogenerated).

**Describe the solution you'd like**
The image description should be empty by default.
The file name could still be displayed as a hint if no description is set, but not be set as the value.

**Describe alternatives you've considered**
In my use case it would make sense if the image description is the nearest heading by default, but i assume that is not useful for everyone.

**Comment by juliushaertl** [comment]
Sounds like a good idea. In addition I'd be curious if we can also preserve the filename.

**Comment by provokateurin** [comment]
I paste my images from my screenshot app, so the file name for those might just always be image.png 🤷‍♀️. But yeah in case you add an image using drag and drop it would be nice to keep the name

**Comment by jancborchardt** [comment]
Yeah, good point – with a lot of photos (IMG_0001.jpg and such) it is useless and as mentioned in the other issue also an accessibility problem.

As a quick fix I like the proposal of instead leaving it empty and via label encouraging people to fill it out.

For an improved experience we could also do it like Twitter and don't show an input field by default, but encourage people to set alt texts. I think @marcoambrosini also mentioned this in a design review.

**Comment by adi-gupta2** [comment]
Hi @provokateurin can you please assign me this issue. I ll try and solve it. Thanks.

**Comment by provokateurin** [comment]
@adi-gupta2 that's not necessary, you can just work on it and create a PR when done
